bundle/manifests/mcg-osd-deployer.clusterserviceversion.yaml

```diff
@@ -36,6 +37,61 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
```

Comment:
please avoid adding more apiGroups in cluster permission, I have added comment in @v-harihar PR

Reply:
Removed.